### DISCUSSION TITLE
replacements in custom commit message

### DISCUSSION THREAD

**Discussion by return42** [JamesIves/github-pages-deploy-action#681] [Q&A] [answered]
Is there a way to have replacements in a custom commit message like:

```yml
     COMMIT_MESSAGE: "build from commit ${{ process.env.GITHUB_SHA }}" 
```

**Comment by return42** [comment, marked as answer]
I had to use:

```yaml
        COMMIT_MESSAGE: build from commit ${{ github.sha }}
```

**Reply by JamesIves** [reply]
Glad you figured this out. I'll probably add a part in the README about this as this question has come up before, cheers.

**Reply by return42** [reply]
What I didn't know when I asked the question : I must not use the process environment context, I had to use [github action context](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context) / I'm not familiar with github actions :-)
